beacon-chain/rpc/validator_server.go

```diff
@@ -226,3 +218,23 @@ func (vs *ValidatorServer) retrieveActiveValidator(beaconState *pbp2p.BeaconStat
 	}
 	return beaconState.ValidatorRegistry[validatorIdx], nil
 }
+
+func (vs *ValidatorServer) retrieveActiveValidatorsPublicKeys(pubkeys [][]byte) ([][]byte, error) {
```

Comment:
Please add a comment to this method since it's a bit hard to explain what it does in the method name. 

This method takes a list of validator public keys, filters the active ones, and returns the list of active public keys. Maybe this method is better named "filterActivePublicKeys"? Up to you, but it isn't intuitive as is

Reply:
chose the filter naming